src/clients/java/docs.zig

```diff
@@ -92,21 +92,16 @@ pub const JavaDocs = Docs{
     .install_prereqs = "apk add -U maven openjdk11",
 
     .current_commit_pre_install_commands = 
-    \\mvn deploy:deploy-file -Durl=file://$(pwd) -Dfile=$(find . -name '*-SNAPSHOT.jar') -DgroupId=com.tigerbeetle -DartifactId=tigerbeetle-java -Dpackaging=jar -Dversion=0.0.1-3431
+    \\mvn deploy:deploy-file -Durl=file://$SAMPLE_ROOT -Dfile=$(find . -name '*-SNAPSHOT.jar') -DgroupId=com.tigerbeetle -DartifactId=tigerbeetle-java -Dpackaging=jar -Dversion=0.0.1-3431
     \\mkdir -p $HOME/.m2/
-    \\echo '<settings xmlns="http://maven.apache.org/SETTINGS/1.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/SETTINGS/1.0.0 http://maven.apache.org/xsd/settings-1.0.0.xsd"><localRepository>'$(pwd)'</localRepository></settings>' > $HOME/.m2/settings.xml
+    \\echo '<settings xmlns="http://maven.apache.org/SETTINGS/1.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/SETTINGS/1.0.0 http://maven.apache.org/xsd/settings-1.0.0.xsd"><localRepository>'$SAMPLE_ROOT'</localRepository></settings>' > $HOME/.m2/settings.xml
```

Comment:
It's ok for a CI runner, but overwriting the `$HOME/.m2/settings.xml` file in a developer machine seems odd. We should instead specify it on maven command line `mvn --global-settings PATH_TO_ALTERNATIVE_SETTINGS_XML`.

Reply:
Yeah I agree this is not good. I even use `mvn -s local-settings.xml` elsewhere. I will remove this but not necessarily in this PR.

Reply:
I ended up making this change in this pr.